src/test/java/core/basesyntax/service/DataTransactionServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.DataTransactionServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataTransactionServiceImplTest {
+
+    private static DataTransactionService dataTransaction;
+    private final String dataFromInput = new StringBuilder("type,fruit,quantity")
+            .append(System.lineSeparator())
+            .append("b,banana,20").append(System.lineSeparator())
+            .append("b,apple,100").append(System.lineSeparator())
+            .append("s,banana,100").append(System.lineSeparator())
+            .append("p,banana,13").toString();
+
+    @BeforeClass
+    public static void setUp() {
+        dataTransaction = new DataTransactionServiceImpl();
+    }
+
+    @Test
+    public void dataTransaction_parseData_ok() {
```

Comment:
```suggestion
    public void turnDataToTransactions_parseData_ok() {
```
first part is the name of the method that you test, fix it in every test method of every test class